spaceship/lib/spaceship/connect_api/token.rb

```diff
@@ -100,7 +100,8 @@ def refresh!
 
         payload = {
           iss: issuer_id,
-          iat: now.to_i,
+          # reduce the issued-at-time in case our time is slighly ahead of apple's servers, which causes the token to be rejected
```

Comment:
```suggestion
          # Reduce the issued-at-time in case our time is slighly ahead of Apple's servers, which causes the token to be rejected.
```